client/src/dashboard/pages/contacts/Contacts.jsx

```diff
@@ -37,6 +39,21 @@ function Contacts() {
     { title: 'Extension', field: 'exten' },
   ];
 
+  const filteredContacts = contacts.filter((contact) => {
```

Comment:
Not something that needs to be done but a future consideration is to make this a reusable function.

import { filterItemsBySearch } from ../util 
const filteredContacts = contacts.filter(() => filterItemsBySearch(searchTerm))